src/hub/hubUri.ts

```diff
@@ -22,8 +23,8 @@ export default async function hubUri(
     );
   }
 
-  if (path) {
-    hubUrl.pathname += path.startsWith("/") ? path : `/${path}`;
+  if (hubPath) {
+    hubUrl.pathname = path.join(hubUrl.pathname, hubPath);
```

Comment:
That url path or fs path? If it's url, I think it's not the right lib to use

Reply:
Path is a path is a path

Reply:
nope https://stackoverflow.com/a/16301947/5520074

Reply:
Because it won't work as expected in windows for example

Reply:
This is not URL, it is a join between two paths, hubUrl.pathname and hubPath

Reply:
You are joining url paths. The path separator is \ in windows, so it will be miss behaving

Reply:
Never mind the VSCODE extensions can't use this package anyway